src/libraries/System.Runtime.InteropServices/gen/ComInterfaceGenerator/ComInterfaceGenerator.cs

```diff
@@ -680,7 +682,7 @@ private static ClassDeclarationSyntax GenerateInterfaceInformation(ComInterfaceC
                     FieldDeclaration(VariableDeclaration(VoidStarStarSyntax, SingletonSeparatedList(VariableDeclarator(vtableFieldName))))
                         .AddModifiers(Token(SyntaxKind.PrivateKeyword), Token(SyntaxKind.StaticKeyword)),
                     // public static void* VirtualMethodTableManagedImplementation => _vtable != null ? _vtable : (_vtable = InterfaceImplementation.CreateManagedVirtualMethodTable());
-                    PropertyDeclaration(PointerType(PredefinedType(Token(SyntaxKind.VoidKeyword))), "VirtualMethodTableManagedImplementation")
+                    PropertyDeclaration(PointerType(PointerType(PredefinedType(Token(SyntaxKind.VoidKeyword)))), "ManagedVirtualMethodTable")
```

Comment:
We have a VoidStarStar member that pre-constructs the type syntax. Let's use that for consistency.